pandas/tests/test_style.py

```diff
@@ -164,6 +164,13 @@ def test_multiindex_name(self):
 
         self.assertEqual(result['head'], expected)
 
+    def test_numeric_columns(self):
+        # https://github.com/pydata/pandas/issues/12125
+        df = pd.DataFrame({0: [1, 2, 3]})
+        df.style._translate()
+
+        self.assertTrue(True)
```

Comment:
no just put a comment that this is a smoke test

if an exception is raised the test fails